src/gamelogic/gamemodel/Environment.hpp

```diff
@@ -56,80 +56,86 @@ namespace GameModel {
             void shuffleCardPile();
 
             /**
-             *
-             *
-             * @return
+             * increment the election tracker by one
+             * @return the actual value of the election tracker
```

Comment:
```suggestion
             * @return the current value of the election tracker
```
actual=tatsächlich, current=aktuell